src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -776,9 +776,11 @@ public async Task Edit()
                     var permissions = FormatPermissions(newCertificate);
 
                     LogAuditFor("Certificates", "CHANGE", 
-                        $"Certificate {newCertificate?.Name}. Security Clearance: {newCertificate?.SecurityClearance}. Permissions: {permissions}. TwoFactor: {string.IsNullOrEmpty(twoFactorAuthenticationKey) == false}");
+                        $"Certificate {newCertificate?.Name}. Security Clearance: {newCertificate?.SecurityClearance}. Permissions: {permissions}. TwoFactor: {string.IsNullOrEmpty(twoFactorAuthenticationKey) == false}. Not After: {newCertificate?.NotAfter}.)");
                 }
 
+                var notAfter = newCertificate.NotAfter ?? existingCertificate.NotAfter;
```

Comment:
Since the endpoint is "edit" naming here is confusing. It's not a `new` certificate - it's the same one with some properties updated. Would be better if we said "updated" or "edited".

Also I don't quite understand how can you change a not after on the existing certificate? @ppekrol 
I have a feeling either the "edit" is more than edit only or I don't understand something.

Reply:
I can change the name to `editedCertificate`. I don't know how this EP should work. The only thing I've done is edit the NotAfter field for the purpose of regenerate a certificate.

Reply:
@ppekrol can you please explain what are the responsibilities of the edit endpoint?

Reply:
we can change the name if you want, not a problem. Few lines up we are doing:

```
                    var existingCertificateJson = ServerStore.Cluster.GetCertificateByThumbprint(ctx, newCertificate.Thumbprint);
                    if (existingCertificateJson == null)
                        throw new InvalidOperationException($"Cannot edit permissions for certificate with thumbprint '{newCertificate.Thumbprint}'. It doesn't exist in the cluster.");

```

so it is an edit, since the needs to be an existing certificate